### ISSUE TITLE
Shading Bug 1.7.10

### ISSUE THREAD

**Issue by FlailingSpade** [Team-Fruit/SignPicture#53]
Forge Version: 10.13.4.1558
SignPicture Version: 2.8.0

Placing a signpicture in the world causes chests (and similar items) to lose their shading, particularly items that are placed _after_ the signpicture was already placed. I first noticed this bug when placing objects from the Decocraft mod, however I am able to reproduce it with only SignPicture installed using vanilla chests. The easiest way I've found to reproduce this is to place down a few chests, then place a signpicture, and refresh with F3+A. This may be the same issue as #49 

I know I am using a very outdated version (I am building a world using some mods that have not been updated past 1.7.10) so I really don't expect a special update to such an old version to fix this issue. However, I am making this post with the hope that there may be something on my end I can do to fix the problem. Any help is appreciated!

Chests before placing sign
![beforesign](https://user-images.githubusercontent.com/9277094/74019355-7b74d400-4965-11ea-9e6c-2b7f47e5e795.png)

After placing sign
![aftersign](https://user-images.githubusercontent.com/9277094/74019361-7d3e9780-4965-11ea-825a-c7f4122c110c.png)

**Comment by GedeonGrays** [comment]
I confirm the problem.
Ran through all the modpack (165 mods) and sure enough, this glitch with glowing tiles (it affects not only chests, but a bunch of other blocks (including mods) caused SignPicture.
Please, could you (@Kamesuta) fix this glitch. It is quite stressful during the game and distracts from the gameplay.

**Comment by sousuke0422** [comment]
Unfortunately, this mod is not actively being updated at this time. It may be merged if someone makes a pull request, but who knows.

---

残念ながら現在このModは積極的な更新は行われていません。誰かがpull requestを行えばマージされるかもしれませんがどうなるかはわかりません。

**Comment by GedeonGrays** [comment]
Of course, it's sad to read this, but I understand you. We'll have to find a solution to the problem ourselves.
In any case, thank you for your quick response.
